V11_SPECIFICATION.md

```
@@ -0,0 +1,132 @@
+
+---
+
+## 3. Extended State Vector
+
+The state vector is extended from 10 to 13 dimensions:
+
+`y = [ψ, φ, ω, ε, E, K, Γ, I_ω, ΔΦ_acc, ψ_mem, ∂Ω/∂Ψ, ΘN, Λ]`
+
+| Index | Symbol | Variable | Description |
+|:---:|:---:|:---|:---|
+| 0-9 | (existing) | Original 10 state variables from v10.0 |
```

markdownlint-cli2 (0.18.1)

Table column count
Expected: 4; Actual: 3; Too few cells, row will be missing data

(MD056, table-column-count)